src/pages/workspaces/workspace/analysis/AppLauncher.js

```diff
@@ -152,11 +152,11 @@ const ApplicationLauncher = _.flow(
 
       const proxyUrl = runtime?.proxyUrl
       const url = await Utils.switchCase(application,
-        [tools.terminal.label, () => `${proxyUrl}/terminals/1`],
-        [tools.spark.label, () => getSparkInterfaceSource(proxyUrl)],
-        [tools.RStudio.label, () => proxyUrl],
-        [tools.Azure.label, () => `${proxyUrl}/lab`],
-        [Utils.DEFAULT, () => console.error(`Expected ${application} to be one of terminal, spark, ${tools.RStudio.label}, or ${tools.Azure.label}.`)]
+        [toolLabels.terminal, () => `${proxyUrl}/terminals/1`],
+        [toolLabels.spark, () => getSparkInterfaceSource(proxyUrl)],
+        [toolLabels.RStudio, () => proxyUrl],
+        [toolLabels.JupyterLab, () => `${proxyUrl}/lab`],
+        [Utils.DEFAULT, () => console.error(`Expected ${application} to be one of terminal, spark, ${toolLabels.RStudio}, or ${toolLabels.JupyterLab}.`)]
```

Comment:
Is this message still correct with JupyterLab instead of Azure?

Reply:
Yes, Jupyterlab can be proxied to from this page.